src/components/SecondaryPanes/Expressions.css

```diff
@@ -66,20 +69,32 @@
 
 .expression-container {
   border: 1px;
-  padding: 0.6em 1em 0.6em 0.5em;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  padding-inline-start: 20px;
+  padding-inline-end: 12px;
   width: 100%;
   color: var(--theme-body-color);
   background-color: var(--theme-body-background);
   display: block;
   position: relative;
-  min-height: var(--breakpoint-expression-height);
 }
 
 .expression-container > .tree {
   width: 100%;
   overflow: hidden;
 }
 
+.expression-container .tree .tree-node[aria-level="1"] {
+  padding-top: 0px;
+}
+
+.expression-container .tree-node[aria-level="1"] .object-label {
+  font-family: var(--monospace-font-family);
+  line-height: 13px;
+  padding-top: 1px;
```

Comment:
Not a fan of this kind of pixel-shifting (the line-height + padding combo). It tends to work for you on one browser with one font at one zoom level, and not so much in other conditions.

Maybe we could use a 14px or even 16px line-height for all text in those tree nodes?

Reply:
Took this out and set the line-height to 14px instead.